cmd/ci-secret-bootstrap/bitwarden/cli.go

```diff
@@ -0,0 +1,24 @@
+package bitwarden
+
+type bitwardenCliClient struct {
+}
+
+func (c *bitwardenCliClient) Login(username, passward string) error {
+	return nil
+}
+
+func (c *bitwardenCliClient) ListItems() ([]Item, error) {
+	return []Item{}, nil
+}
+
+func (c *bitwardenCliClient) GetAttachment(itemID, AttachmentID string, raw bool) (string, error) {
+	return "", nil
+}
+
+func (c *bitwardenCliClient) Logout() error {
+	return nil
+}
+
+func newBitwardenCliClient() *bitwardenCliClient {
```

Comment:
Return the interface, not the type, and export the constructor

Reply:
Not sure if I did as you expected. Please check the current version.